README.md

```diff
@@ -7,367 +7,506 @@
 [![Documentation badge](https://img.shields.io/readthedocs/iotagent-node-lib.svg)](http://iotagent-node-lib.readthedocs.org/en/latest/?badge=latest)
 ![Status](https://nexus.lab.fiware.org/static/badges/statuses/iot-node-lib.svg)
 
-## Index
-
-* [Overview](#overview)
-  + [Description](#description)
-  + [Device to NGSI Mapping](#device-to-ngsi-mapping)
-  + [Features](#features)
-  + [The `TimeInstant` element](#the-timeinstant-element)
-  + [Implementation decisions](#implementation-decisions)
-* [Build and Install](#build-and-install)
-* [API Overview](#api-overview)
-  + [About API](#about-api)
-  + [Device Provisioning API](#device-provisioning-api)
-  + [Configuration API](#configuration-api)
-* [Advanced Topics](#advanced-topics)
-  + [Secured access to the Context Broker](#secured-access-to-the-context-broker)
-  + [Data mapping plugins](#data-mapping-plugins)
-  + [Old IoTAgent data migration](#old-iotagent-data-migration)
-* [Testing](#testing)
-  + [Agent Console](#agent-console)
-  + [Agent tester](#agent-tester)
-* [Development documentation](#-development-documentation)
-
-## Overview
-### Description
-This project aims to provide a Node.js module to enable IoT Agent developers to build custom agents for their devices that can
-easily connect to NGSI Context Brokers (such as [Orion](https://github.com/telefonicaid/fiware-orion) ).
+This project aims to provide a Node.js module to enable IoT Agent developers to
+build custom agents for their devices that can easily connect to NGSI Context
+Brokers (such as [Orion](https://github.com/telefonicaid/fiware-orion) ).
```

Comment:
Whitespace should be removed before last `)`.

Reply:
Fixed - e12947e